packages/sdk/src/reads/getTheGraphDelegationPool.ts

```diff
@@ -0,0 +1,22 @@
+import { type ReadContractParameters, readContractParameters } from "../utils/viem.js";
+import { type Address, type PublicClient, parseAbi } from "viem";
+
+const abiStakingContract = parseAbi([
+  "function delegationPools(address indexer) view returns (uint32 cooldownBlocks, uint32 indexingRewardCut, uint32 queryFeeCut, uint256 updatedAtBlock, uint256 tokens, uint256 shares)",
+] as const);
```

Comment:
This is an impure side effect that can be avoided (function call at root scope). Please translate this into JSON (we do that elsewhere too).

Reply:
done